traceKeyExtractor.py

```diff
@@ -222,10 +189,10 @@ def discoverKeys():
 #Run the mf_nonce_brute program with the provided arguments to decode a key
 #Returns a key on success, "" otherwise
 def bruteForce(args):
-    cmd_list = [mfNonceBruteCommand] + args
+    cmd_list = [pm3Location / mfNonceBruteCommand] + args
     print("    Running bruteforce command:")
     print("    ",end = "")
-    print(" ".join(cmd_list))
+    print(f"tools/mf_nonce_brute {" ".join(cmd_list[1:])}")
```

Comment:
```suggestion
    print(f"tools/mf_nonce_brute {' '.join(cmd_list[1:])}")
```
Fixes syntax error when double quotes are used inside double quotes.